wdtk-client/src/main/java/org/wikidata/wdtk/client/DumpProcessingAction.java

```diff
@@ -29,16 +29,16 @@
  * {@link EntityDocumentDumpProcessor}. Additional methods provide a generic
  * interface for setting options and other auxiliary information that the action
  * might require.
- *
+ * 
  * @author Markus Kroetzsch
- *
+ * 
  */
 public interface DumpProcessingAction extends EntityDocumentDumpProcessor {
 
 	/**
 	 * Returns true if processing requires a {@link Sites} object to be set.
 	 * This is the case for some operations that process site links.
-	 *
+	 * 
```

Comment:
Could you try to switch this off in your Eclipse? When I safe a file, it removes these spaces. The formatter (SHIFT+CTRL+F) does not change the spaces in my installation. As it is now, we are getting a lot of distracting changes in the diffs based on this.